src/test/url-handling.test.js

```diff
@@ -1178,7 +1178,7 @@ describe('url upgrading', function() {
           pathname:
```

Comment:
In this file, I believe we should also update the `selectedThread` block above:
* add v=${CURRENT_URL_VERSION}
* add a test for selecting several tracks

It would probably be good to add a few testcases starting with `_getStoreWithURL` with the latest version (so not specifying any) but that's optional because we don't have any yet.

Reply:
I've made those two changes. I haven't added any tests with `_getStoreWithURL` and I'm not exactly sure what kinds of tests you'd like to see.
In my opinion, we shouldn't add too many tests that need to know the serialized form of the state, because those tests need to be updated any time we change the URL serialization. I think it would be better to write tests that dispatch actions to change the state, and then make sure this state is persisted in the URL, without testing what the URL looks like. Only by checking the state, via selectors.

Reply:
Yeah I agree with that.

However I believe it's still good to have a few tests showing the URL serializing, so that the expectations about the serialized format is explicit.

Also starting with a URL is necessary to test that we're handling properly unwanted inputs :-)

Reply:
That makes sense.